jasmine/index.d.ts

```diff
@@ -113,18 +124,14 @@ declare namespace jasmine {
         withMock(func: () => void): void;
     }
 
-    interface CustomEqualityTester {
-        (first: any, second: any): boolean;
-    }
+    type CustomEqualityTester = (first: any, second: any) => boolean;
```

Comment:
I have not used `CustomEqualityTester` myself so far, so I'm unsure about its usage.

In the example it doesn't make much sense as the function tests for the type of the parameter during runtime to do different  equality tests, so `any` would be the best type here *any*way (no pun intended 😄 )

If both parameters to compare are of a different type, we would need two generic parameters which would complicated matter quickly

If this makes sense in real world usage, I would  add it in a different PullRequest